arm_wiki/Alternate-Install-TrueNAS.md

```diff
@@ -0,0 +1,63 @@
+# IMPORTANT
```

Comment:
I'm a bit confused about this note. We're installing a docker container, but then the note says it's not supported and to... use... Docker instead?
Is that just a copy paste from the others?